test/e2e/security_test.go

```diff
@@ -169,30 +214,43 @@ func testCert(t *testing.T, data *TestData, expectedCABundle string, restartPod
 			t.Log("CA bundle doesn't match the expected one, retrying")
 			return false, nil
 		}
-		clientConfig := restclient.Config{
-			TLSClientConfig: restclient.TLSClientConfig{
-				Insecure:   false,
-				ServerName: certificate.GetAntreaServerNames(certificate.AntreaServiceName)[0],
-				CAData:     []byte(caBundle),
+		cm := corev1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "ConfigMap",
+				APIVersion: "v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "antrea-ca-e2e",
+				Namespace: testNamespace,
```

Comment:
Please don't forget to check the github action results, it shows 
`undeclared name: 'testNamespace' (typecheck)`

Reply:
I am wondering have you verified the change locally? you didn't even set `testNamespace`.